<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColoringDialog</class>
 <widget class="QDialog" name="ColoringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>385</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="scheduleGrd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section {
    background-color: rgb(11, 117, 250);
    height: 32px;
	border-bottom-style: dotted;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="coursesCmbBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cursos</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblLoad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 14px;</string>
    </property>
    <property name="text">
     <string>Cantidad de bloques diarios:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessing">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>381</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="colorBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
      background-color: rgb(230, 19, 249);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }

QPushButton:hover:!pressed
{
      background-color: rgb(189, 0, 174);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}
QPushButton:disabled {
background-color:rgb(136, 138, 133);
}</string>
       </property>
       <property name="text">
        <string>Colorear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
      background-color: rgb(250, 78, 78);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }


QPushButton:hover:!pressed
{
      background-color: rgb(204, 0, 0);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
